Top 5 rated movies

MagnusTerak opened the pull request:
**Description**

This PR creates the section Top 5 Rated movies on top of the startpage.
---

**Features made in the pull-request**
* Added new section (see image)
* Functions added to retrieve the top 5 rated movies from the API
* New API connection "/movies/top-rated-movies" to access these 5 movies. 

---

**Changes made in the pull-request**
* onGoingMovies.js & scss has been modified and renamed to filtering.js & scss to remove the movie section and still keep the filtering
* new files created for top5rated
---

**Testing**
Created 2 tests that testes the required functions of this "section".
Ran all tests and all are working fine


![image](https://github.com/user-attachments/assets/c8e13fae-3bd6-4d1a-aee6-d4c31cee8920)

nilsishome reviewed (approved):
Great job with getting the functionality right in your new addition to our API!
Your code makes sense and is easy to follow, and your tests are very detailing and passing, which is also great!
## Nice work, Magnus!

Tarupotter reviewed (commented):
It looks really good, however, there are a few things I can mention.
The movie posters seem to be a little different in size, 
also, i cant find why this line of code is included: 
_const movieContainer = document.querySelector(".filtering");_ in topRatedMovies.js
 Otherwise, great job!

MagnusTerak commented:
> It looks really good, however, there are a few things I can mention. The movie posters seem to be a little different in size, also, i cant find why this line of code is included: _const movieContainer = document.querySelector(".filtering");_ in topRatedMovies.js Otherwise, great job!

Thank you for your review, great eye you got there, the stress got into me and i missed these "small" things. They are now adjusted. the movieContainer const was part of old code that i removed but i forgot that line, but it is now removed!
